src/website/views/components/drawer/_examples.pug

```diff
@@ -782,6 +782,79 @@ ul#chi-tabs--header.chi-tabs
         <!-- JavaScript -->
         <script>chi.drawer(document.getElementById('drawer-trigger-21'));</script>
 
+h3 Portal
+p.-text
+  | Header markup is optional. Remove it to make the content fill the drawer.
```

Comment:
Let's remove this line. No need to replace with other text for now as you've added example descriptions at the tab level.